components/backstage/components/backstage_list.tsx

```diff
@@ -41,7 +59,12 @@ const BackstageList = ({searchPlaceholder = localizeMessage('backstage_list.sear
         children = React.Children.map(children, (child) => {
             return React.cloneElement(child, {filterLowered});
         });
-        if (children.length === 0 || !hasChildren) {
+        length = children.length;
+        if (typeof remainingProps.page != 'undefined') {
```

Comment:
you can just check if `remainingProps.page !== undefined` but preferably we would use the convention of the early return

Reply:
@nevyangelova should've tagged you in this for visibility, but the above explanation is for why i don't early return